src/main/java/edu/hm/hafner/analysis/parser/AquaScannerParser.java

```diff
@@ -0,0 +1,92 @@
+package edu.hm.hafner.analysis.parser;
+
+import java.text.MessageFormat;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+/**
+ * Parser for Aqua Scanner CLI (scannercli) tool.
+ *
+ * @author Juri Duval
+ */
+public class AquaScannerParser extends JsonIssueParser {
+    private static final String VALUE_NOT_SET = "-";
+    private static final String AQUA_VULNERABILITY_LEVEL_TAG_SENSITIVE = "sensitive";
+    private static final String AQUA_VULNERABILITY_LEVEL_TAG_MALWARE = "malware";
+    private static final String AQUA_VULNERABILITY_LEVEL_TAG_CRITICAL = "critical";
+    private static final String AQUA_VULNERABILITY_LEVEL_TAG_HIGH = "high";
+    private static final String AQUA_VULNERABILITY_LEVEL_TAG_MEDIUM = "medium";
+    private static final String AQUA_VULNERABILITY_LEVEL_TAG_LOW = "low";
+    private static final String AQUA_VULNERABILITY_LEVEL_TAG_NEGLIGIBLE = "negligible";
+    private static final long serialVersionUID = 1L;
+
+    @Override
+    protected void parseJsonObject(final Report report, final JSONObject jsonReport, final IssueBuilder issueBuilder) {
+        JSONArray resources = jsonReport.optJSONArray("resources");
+        if (resources != null) {
+            parseResources(report, resources, issueBuilder);
+        }
+    }
+
+    private void parseResources(final Report report, final JSONArray resources, final IssueBuilder issueBuilder) {
+        for (int i = 0; i < resources.length(); i++) {
+            JSONObject resourceWrapper = (JSONObject) resources.get(i);
```

Comment:
The other parsers make sure that the object is of correct type. Does it make sense here as well?
```
if (arrayElement instanceof JSONObject)
```

Reply:
The structure is always JSONObject but there is no harm in making an extra check.
Fixed.